docs/config/README.md

```diff
@@ -327,6 +327,7 @@ it would have been `nixpkgs/pkgs`.
 | `truncate_to_repo`  | `true`        | Whether or not to truncate to the root of the git repo that you're currently in. |
 | `style`             | `"bold cyan"` | The style for the module.                                                        |
 | `disabled`          | `false`       | Disables the `directory` module.                                                 |
+| `prefix`            | `"in "`       | Prefix to display immediately before the directory.                              |
```

Comment:
I think the convention is to put the `prefix` above the `style` and `disabled` variables.

Reply:
Thanks for the note. Just pushed an update to move it up.